wdl/TrainGCNV.wdl

```diff
@@ -102,6 +102,8 @@ workflow TrainGCNV {
     RuntimeAttr? runtime_attr_explode
   }
 
+  Boolean perform_subsampling = defined(n_samples_subsample) && (select_first([n_samples_subsample]) < length(samples))
```

Comment:
I would probably put this right above the check for whether to run RandomSubsampleStringArray since that's where it's used

Reply:
Thanks. Just pushed a new commit that integrates this new variable into the existing if statement that checks whether to run `RandomSubsampleStringArray` - let me know if this is better, or if I should pull it out into a new variable like I'd done here.

Reply:
I don't have strong feelings either way so this looks good to me!